.env.example

```diff
@@ -15,9 +15,14 @@ ZENDESK_TAG='A tag used to route the ticket to the right queue'
 ZENDESK_TAG_ONE_LOGIN_ADMIN_TOOL='A tag used to route the ticket to the Admin tool queue'
 ZENDESK_GROUP_ID='A groupd ID used to assign the ticket to the right group'
 
+# ServiceNow
+SERVICENOW_AUTH_CRDENTIALS='Credentials to access Service Now'
+SERVICENOW_URL='https://onelogingovukdev.service-now.com/api/gds'
```

Comment:
Remove that URL as this is a secret and replace the value in this file with dummy value.

Reply:
Now changed to https://dummyurl